drivers/sdhc/rcar_mmc.c

```diff
@@ -233,13 +233,12 @@ static int rcar_mmc_check_errors(const struct device *dev)
  */
 static int rcar_mmc_poll_reg_flags_check_err(const struct device *dev,
 	unsigned int reg, uint32_t flag, uint32_t state, bool check_errors,
-	bool check_dma_errors)
+	bool check_dma_errors, int32_t timeout_us)
```

Comment:
`timeout` field of `struct sdhc_command` has type `int`.
Removed `remain_us` had type `int`. Why this new parameter is `int32_t`? You need to propagate types from the higher level API if there are not strict reasons to not do so.

Reply:
int32_t and int are the same for Zephyr, take a look how zephyr defines int max:
```
#define INT_MAX		__INT_MAX__
``` 
--->
```
#if __INT_MAX__ == 2147483647
#define __INT_WIDTH__ 32
#define __SIZEOF_INT__ 4
#else
#error "unexpected __INT_MAX__ value"
#endif
```
so, here it is nothing to worry